### PR TITLE
Add local:security:check back

### LINKED ISSUE #45: Re-add `check:security` command?
As Fabien said [here](https://github.com/symfony-cli/symfony-cli/issues/9#issuecomment-1003554277):

> The (check:security) command does not exist in the new version of the CLI, you should use https://github.com/fabpot/local-php-security-checker instead.

Unless there are technical reasons to not do it, could we please reconsider adding the command again? It was very useful and the alternative of having to install another project to have that feature causes lots of friction in my opinion.

Thanks!

### PR DESCRIPTION
**Pull request by fabpot**
Closes #45

### CODE REVIEW

**Comment by fabpot** [comment]
As an added bonus, I've added support for Github action.

File: commands/local_check_security.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	Usage:    "Check security issues in project dependencies",
+	Description: `Checks security issues in project dependencies. Without arguments, it looks
+for a "composer.lock" file in the current directory. Pass it explicitly to check
+a specific "composer.lock" file.`,
+	Flags: []console.Flag{
+		dirFlag,
+		&console.StringFlag{
+			Name:         "format",
+			DefaultValue: "ansi",
+			Usage:        "The output format (ansi, markdown, json, or yaml)",
```
Comment (andrew-demb) [inline commands/local_check_security.go:26]:
```suggestion
			Usage:        "The output format (ansi, markdown, json, junit, or yaml)",
```

File: commands/local_check_security.go [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+		output, err := security.Format(vulns, format)
+		if err != nil {
+			return console.Exit(fmt.Sprintf("unable to output the results: %s", err), 127)
+		}
+		terminal.Stdout.Write(output)
+
+		if os.Getenv("GITHUB_WORKSPACE") != "" {
+			// Ran inside a Github action, export vulns
+			output, _ := security.Format(vulns, "raw_json")
+			terminal.Printf("::set-output name=vulns::%s", output)
```
[the remark this answers is not in the document]
Comment (tucksaun) [reply commands/local_check_security.go:78]: It looks like GitHub action supports ::set-output being written to stderr (I tried that with `echo "::set-output name=foo::bar">&2` in a simple workflow).
So it should be possible to write to stderr here and so supports every use cases possible.

[the remark this answers is not in the document]
Comment (fabpot) [reply commands/local_check_security.go:78]: See #54

**Pull request merged**